python/cuml/dask/naive_bayes/naive_bayes.py

```diff
@@ -51,50 +51,50 @@ class MultinomialNB(BaseEstimator,
 
     .. code-block:: python
 
-    import cupy as cp
+        import cupy as cp
 
-    from sklearn.datasets import fetch_20newsgroups
-    from sklearn.feature_extraction.text import CountVectorizer
+        from sklearn.datasets import fetch_20newsgroups
+        from sklearn.feature_extraction.text import CountVectorizer
 
-    from dask_cuda import LocalCUDACluster
-    from dask.distributed import Client
+        from dask_cuda import LocalCUDACluster
+        from dask.distributed import Client
 
-    from cuml.dask.common import to_sparse_dask_array
+        from cuml.dask.common import to_sparse_dask_array
```

Comment:
Man, I can't wait until this is no longer needed!

Reply:
+1 for sure. Related, Thanks for all the amazing sparse matrix work you have been doing. Should make lives easier all around.